pkgs/games/steam/fhsenv.nix

```diff
@@ -303,6 +299,7 @@ in buildFHSEnv rec {
   '' + args.extraPreBwrapCmds or "";
 
   extraBwrapArgs = [
+    "--bind /etc/NIXOS /etc/NIXOS" # required 32bit driver check in runScript
```

Comment:
Wouldn't this fit better in `etcBindEntries` in the `buildFHSEnv` code?

Reply:
I am not sure if this is something we should universally do. Maybe?

Reply:
Maybe just kill the check instead?

Reply:
It's a good hint for people to properly use the module. We can kill it later if it causes more pain.